sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs

```diff
@@ -49,6 +49,10 @@ public void ValidateMetricTelemetryItem()
             Assert.Equal("TestDoubleCounter", metricsData.Metrics.First().Name);
             Assert.Equal(123.45, metricsData.Metrics.First().Value);
             Assert.Null(metricsData.Metrics.First().DataPointType);
+
+            Assert.Equal(2, metricsData.Properties.Count);
+            Assert.Contains(metricsData.Properties, kvp => kvp.Key == "tag1" && kvp.Value == "value1");
+            Assert.Contains(metricsData.Properties, kvp => kvp.Key == "tag2" && kvp.Value == "1,2,3");
```

Comment:
can you extend it to test an array which is null.

Reply:
I think SDK itself might skip adding Metric.tags.

Reply:
there's already a check in case the entire tag.Value is null.
If the array values are null, we build an empty string.